services/jetbrains/jetbrains-rating.tester.js

```diff
@@ -0,0 +1,84 @@
+'use strict'
+
+const { withRegex, isStarRating } = require('../test-validators')
+const t = (module.exports = require('../tester').createServiceTester())
+
+const isRatingWithReviews = withRegex(/^(([0-4](.?([0-9]))?)|5)\/5$/)
```

Comment:
What does "WithReviews" mean here?

Side note: We should probably have a validator for this in `test-validators` I'm going to raise this as an issue because having a quick look over the other rating tests, this seems to be something we're dealing with a bit inconsistently across services. We don't need to do it in this PR.

Reply:
Probably referenced from the pkgreview tests. I believe the distinction was made in the name to reflect the fact that some services provide the rating data as well as the count of reviews that contributed to that rating, whereas other services do not.

https://img.shields.io/wordpress/plugin/rating/bbpress
![](https://img.shields.io/wordpress/plugin/rating/bbpress)

vs:

https://img.shields.io/amo/rating/dustman
![](https://img.shields.io/amo/rating/dustman)

Reply:
> Probably referenced from the pkgreview tests.

@calebcartwright hit the nail on the head. This was directly lifted from the `pkgreview` tests.

I'll update this to `isRating` to more accurately reflect what I'm checking (not inclusive of 'with reviews') - I'll go ahead and resolve this comment.